lms/djangoapps/email_marketing/tasks.py

```diff
@@ -17,6 +17,27 @@
 SAILTHRU_LIST_CACHE_KEY = "email.marketing.cache"
 
 
+@task(bind=True)
+def get_email_cookies_via_sailthru(self, post_parms):
+    """
+    Adds/updates Sailthru cookie information for a new user.
+     Args:
+        post_parms(dict): User profile information to pass as 'vars' to Sailthru
+    Returns:
+        sailthru_response(dict): cookie fetched from Sailthru
+    """
+    sailthru_client = SailthruClient(email_config.sailthru_key, email_config.sailthru_secret)
+    log.info(
+        'Sending to Sailthru the user interest cookie [%s] for user [%s]',
+        post_parms.get('cookies', ''),
+        user.email
+    )
+
+    sailthru_response = sailthru_client.api_post("user", post_parms)
+
+    return sailthru_response
```

Comment:
But `sailthru_response` is not a primitive. So I am concerned that this object is not serializable.